src/metabase/cache_database.clj

```diff
@@ -0,0 +1,76 @@
+(ns metabase.cache-database
```

Comment:
this should be either "cache-values" or "field-values" or "cached-field-values". any opinions @camsaul ?